client/src/components/Ticketing/ticketingmanager/TicketTypes/tickettypes.tsx

```diff
@@ -0,0 +1,118 @@
+
+import React, {useEffect} from 'react';
+import {DataGrid} from '@mui/x-data-grid';
+import Chip from '@mui/material/Chip';
+import {useAuth0} from '@auth0/auth0-react';
+
+// Web page that manages ticket types
+const TicketTypes = () => {
+  // const [ticketTypes, setTicketTypes] = useState([]);
```

Comment:
We generally shouldn't leave comments without any context like this, but I know this is still getting worked on so it's ok

Reply:
I understand, i did it temporarily just so that there was no warning popup about unused variables.